src/website/webview/models.py

```diff
@@ -15,7 +16,8 @@ class Profile(models.Model):
     """
 
     user = models.OneToOneField(User, on_delete=models.CASCADE)
-    subscriptions = models.ManyToManyField(NixpkgsIssue, related_name="subscribers")
+    channels = models.ManyToManyField(NixChannel, related_name="subscribers")
```

Comment:
@balsoft suggested to actually have something like "collections" or "profiles" (see https://github.com/Nix-Security-WG/nix-security-tracker/issues/532#issuecomment-2851394198), which I think is a good idea if we eventually want users to be able to subscribe programmatically from clients on different machines/setups. Therefore how about we put this into a container?